# Differential Drive Wheel Velocity
# Units should be in radians/s
float64 left_velocity
float64 right_velocity